packages/app-storage/src/Query.tsx

```diff
@@ -107,17 +109,20 @@ class Query extends React.PureComponent<Props, State> {
     return (
       <div className='storage--Query storage--actionrow'>
         <div className='storage--actionrow-value'>
+          {this.keyToName(key)}
+          <hr/>
```

Comment:
Not 100% on-board with this layout. And it is probably 100% to-do with my updated label placement which used to be on-top, but is now on the side. I believe it would be more useful having it on a single line with something like -

`system.accountNonce(5HYmsxGRAmZMjyZYmf7uGPL2YDQGHEt6NjGrfUuxNEgeGBRN): Index`

or even 

`system.accountNonce(AccountId: 5HYmsxGRAmZMjyZYmf7uGPL2YDQGHEt6NjGrfUuxNEgeGBRN): Index`

In the above we will probably start running into some issues anyway, so we will probably need to forgo the label for the output (which is now empty anyway) allow the box to stretch the whole length and then keep the placement of the above where you currently have `Output`

If it becomes messy with the full display (e.g. as per the full address above), give that part a 8rem width and set overflow so it renders something like -

`system.accountNonce(AccountId: 5HYmsxG...): Index`

(And feel free to swap the type or value above)